src/app/organizations/organization-members/organization-members.component.html

```diff
@@ -14,26 +14,28 @@
    limitations under the License.
 -->
 <app-loading [loading]="loading$ | async">
-  <div class="pt-2" fxLayout="column" fxLayoutGap="20px">
+  <div class="pt-2" fxLayout="column" fxLayoutGap="1rem">
     <div *ngFor="let organizationUser of organizationMembers$ | async; let index = index">
-      <mat-card class="m-2" fxFlex *ngIf="organizationUser?.accepted">
-        <div fxLayout="row" fxLayoutGap="10px" fxLayoutAlign="space-between">
-          <div fxFlex fxGrow="3">
-            <mat-card-header>
-              <img *ngIf="organizationUser.user?.avatarUrl" mat-card-avatar class="header-image" [src]="organizationUser.user?.avatarUrl" />
-              <img
-                *ngIf="!organizationUser.user?.avatarUrl"
-                mat-card-avatar
-                class="header-image"
-                src="https://via.placeholder.com/300x300?text=Avatar+Not+Found"
-              />
-              <mat-card-title>{{ organizationUser.user?.username }}</mat-card-title>
-              <mat-card-subtitle>{{ organizationUser.role | titlecase }}</mat-card-subtitle>
-              <mat-card-subtitle *ngIf="organizationUser.user?.orcid">
-                ORCID <img src="../../../assets/images/account-logos/orcid.svg" alt="ORCID iD logo" class="orcid-id-logo mr-3" />
+      <mat-card class="m-2 mat-elevation-z2" fxFlex *ngIf="organizationUser?.accepted">
+        <div fxLayout="row" fxLayoutAlign="space-between stretch">
+          <div fxFlex fxLayout="row" fxLayoutGap="1rem" fxLayoutAlign="center center">
+            <img *ngIf="organizationUser.user?.avatarUrl" mat-card-avatar class="header-image" [src]="organizationUser.user?.avatarUrl" />
+            <img
+              *ngIf="!organizationUser.user?.avatarUrl"
+              mat-card-avatar
+              class="header-image"
+              src="https://via.placeholder.com/300x300?text=Avatar+Not+Found"
+            />
+            <div fxFlex="80" fxLayout="column" fxLayoutAlign="start stretch">
+              <div fxLayout="row" fxLayoutAlign="space-between center">
+                <h5>{{ organizationUser.user?.username }}</h5>
+                <span class="role-chip">{{ organizationUser.role | titlecase }}</span>
```

Comment:
rename to something that doesn't have the word "chip" in it because it's already defined as https://material.io/components/chips#usage which can get confusing . wasn't there some other name used previously? badge (also kind of defined), etc.

this comment applies to all occurences of "chip" when it's not a material chip